xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/common/LedgerIndex.java

```diff
@@ -45,8 +64,13 @@ public static LedgerIndex of(String value) {
    * @param value An {@link UnsignedLong} specifying a ledger index.
    *
    * @return A {@link LedgerIndex} with the given value as a {@link String}.
+   *
+   * @throws NullPointerException if value is null
    */
   public static LedgerIndex of(UnsignedLong value) {
+    if (value == null) {
```

Comment:
You could replace this with `Object.requireNonNull(value)`, which may make Codecov happy

Reply:
Yes, also see comment above. As a style, this should be: `Objects.requireNonNull(value);`

Reply:
done